web3/providers/eth_tester/defaults.py

```diff
@@ -198,8 +198,11 @@ def personal_send_transaction(eth_tester: "EthereumTester", params: Any) -> HexS
     },
     'net': {
         'version': static_return('1'),
-        'peerCount': static_return(0),
+        'peer_count': static_return(0),
```

Comment:
This should still be `peerCount`, since this is meant to mirror the JSON_RPC response that would be issued from a client, and we also wouldn't need the deprecated comment on line 203 either.

Reply:
got a bit too carried away..